db/migrate/20181219094432_add_starting_longitude_and_latitude_to_streams.rb

```diff
@@ -0,0 +1,16 @@
+class AddStartingLongitudeAndLatitudeToStreams < ActiveRecord::Migration
+  def up
+    unless column_exists?(:streams, :start_point_longitude)
+      add_column :streams, :start_point_longitude, :decimal, :precision => 12, :scale => 9
+    end
+
+    unless column_exists?(:streams, :start_point_latitude)
+      add_column :streams, :start_point_latitude, :decimal, :precision => 12, :scale => 9
+    end
+  end
+
+  def down
+    remove_column :streams, :start_point_longitude
+    remove_column :streams, :start_point_latitude
```

Comment:
Does `point` add meaning to the name? Have you considered `start_longitude` and `start_latitude`? Or maybe `first_*`. I'm fine with whatever you choose ;)